### PR TITLE
Rename 'GenerateIdentifiableKey' helper to 'GenerateBase64Key'

### PR DESCRIPTION
**Pull request by marmegh**
### Description
Directory and Project names do not match the overall repository purpose. Additionally, not all coding standards were not fully adhered to on setup of this project, some of those are addressed as part of the renaming effort of this pull request. Addresses issue #9.

### Changes
Updated project/directory from (Tests.)Marvin to (Tests.)Microsoft.Security.Utilities.
Updated method name for GenerateIdentifiableKey to GenerateBase64Key.
Added missing copyright headers to test and class files.
Rearranged using statements where needed to conform to coding guidelines.

### CODE REVIEW

File: src/Microsoft.Security.Utilities/IdentifiableSecrets.cs [hunk 1]
```diff
@@ -16,7 +19,7 @@ public static class IdentifiableSecrets
         /// <param name="keyLengthInBytes">The size of the secret.</param>
         /// <param name="base64EncodedSignature">The signature that will be encoded in the identifiable secret.</param>
         /// <returns></returns>
-        public static string GenerateIdentifiableKey(ulong checksumSeed,
+        public static string GenerateBase64Key(ulong checksumSeed,
                                                      uint keyLengthInBytes,
                                                      string base64EncodedSignature)
```
Comment (eddynaka) [inline src/Microsoft.Security.Utilities/IdentifiableSecrets.cs:24]: pls, fix the indentation

**Comment by eddynaka** [comment]
Change looks good.
Can you also update the title of the PR, pls?

**Review by michaelcfanning — APPROVED** [review]
:shipit:

**Comment by michaelcfanning** [comment]
We forgot the release notes update on this one. Every method name change is a breaking change for customers. I will submit a  new PR with this, stay tuned.

---
In reply to: [1013659836](https://github.com/microsoft/security-utilities/pull/10#issuecomment-1013659836) [](http://example.com/codeflow?ancestors=1013659836,1013659836)

**Pull request merged**